app/ai/itinerary_nodes/intent.py

```
@@ -0,0 +1,81 @@
+            if n is not None and 1 <= n <= 21:
+                state["nights"] = n
+            else:
+                state.pop("nights", None)
+        if (ints := data.get("interests")):
+            state["interests"] = ints
+        if (pace := data.get("pace")):
+            state["pace"] = pace
+        if (cur := data.get("currency")):
+            state["currency"] = cur
+    except Exception:
+        pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)